jisantuc/AFDG
# Add state toggles and width targeting with flex-basis

Overview
-----

This PR adds some minimal state control to test out how different grid interactions work and adds `flex-basis` for responsive design for the 4x4 grid.

Testing
-----

- load the app locally
- resize the window and note that unless you make the window pretty small the rows are preserved
- mess with the state buttons and confirm that behavior is about what you'd expect for a grid with no walls
